[
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/177363801",
    "html_url": "https://github.com/jupyter/notebook/issues/1030#issuecomment-177363801",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1030",
    "id": 177363801,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE3NzM2MzgwMQ==",
    "user": {
      "login": "Carreau",
      "id": 335567,
      "node_id": "MDQ6VXNlcjMzNTU2Nw==",
      "avatar_url": "https://avatars.githubusercontent.com/u/335567?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/Carreau",
      "html_url": "https://github.com/Carreau",
      "followers_url": "https://api.github.com/users/Carreau/followers",
      "following_url": "https://api.github.com/users/Carreau/following{/other_user}",
      "gists_url": "https://api.github.com/users/Carreau/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/Carreau/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/Carreau/subscriptions",
      "organizations_url": "https://api.github.com/users/Carreau/orgs",
      "repos_url": "https://api.github.com/users/Carreau/repos",
      "events_url": "https://api.github.com/users/Carreau/events{/privacy}",
      "received_events_url": "https://api.github.com/users/Carreau/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-01-31T02:37:06Z",
    "updated_at": "2016-01-31T02:37:06Z",
    "author_association": "MEMBER",
    "body": "can you open the developers tools of you browser and looks at any error you would have. \n\nJust the look at the command palette hints that something is wrong with the notebook. \n",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/177911569",
    "html_url": "https://github.com/jupyter/notebook/issues/1030#issuecomment-177911569",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1030",
    "id": 177911569,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE3NzkxMTU2OQ==",
    "user": {
      "login": "AGS-Knight",
      "id": 2307497,
      "node_id": "MDQ6VXNlcjIzMDc0OTc=",
      "avatar_url": "https://avatars.githubusercontent.com/u/2307497?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/AGS-Knight",
      "html_url": "https://github.com/AGS-Knight",
      "followers_url": "https://api.github.com/users/AGS-Knight/followers",
      "following_url": "https://api.github.com/users/AGS-Knight/following{/other_user}",
      "gists_url": "https://api.github.com/users/AGS-Knight/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/AGS-Knight/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/AGS-Knight/subscriptions",
      "organizations_url": "https://api.github.com/users/AGS-Knight/orgs",
      "repos_url": "https://api.github.com/users/AGS-Knight/repos",
      "events_url": "https://api.github.com/users/AGS-Knight/events{/privacy}",
      "received_events_url": "https://api.github.com/users/AGS-Knight/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-02-01T10:52:49Z",
    "updated_at": "2016-02-01T10:52:49Z",
    "author_association": "NONE",
    "body": "This is the console log output when trying to use command palette.\n\n```\n[Log] Default extension for cell metadata editing loaded. (main.min.js, line 24082)\n[Log] Raw Cell Format toolbar preset loaded. (main.min.js, line 24168)\n[Log] Slideshow extension for metadata editing loaded. (main.min.js, line 24216)\n[Warning] actions jupyter-notebook:find-and-replace does not exist, still binding it in case it will be defined later... (main.min.js, line 28971)\n[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (jquery.min.map, line 0)\n[Log] Session: kernel_created (5da56293-4d43-4939-b1f7-d5f8a5e17c01) (main.min.js, line 23370)\n[Log] Starting WebSockets: ws://localhost:8888/api/kernels/ae9cdd62-4425-4776-85e1-b11b12a72faa (main.min.js, line 22673)\n[Log] Kernel: kernel_connected (ae9cdd62-4425-4776-85e1-b11b12a72faa) (main.min.js, line 22329)\n[Log] Kernel: kernel_ready (ae9cdd62-4425-4776-85e1-b11b12a72faa) (main.min.js, line 22329)\n[Log] Loading extension: widgets/notebook/js/extension (main.min.js, line 12109)\n[Log] loaded widgets (extension.js, line 81)\n[Log] Loading extension: IPython-notebook-extensions-3.x/usability/runtools/main (main.min.js, line 12109)\n[Error] Error\n    check (require.js, line 900)\n    (anonymous function) (require.js, line 1140)\n    (anonymous function) (require.js, line 131)\n    (anonymous function) (require.js, line 1190)\n    each (require.js, line 56)\n    emit (require.js, line 1189)\n    check (require.js, line 940)\n    (anonymous function) (require.js, line 1140)\n    (anonymous function) (require.js, line 131)\n    (anonymous function) (require.js, line 1190)\n    each (require.js, line 56)\n    emit (require.js, line 1189)\n    check (require.js, line 940)\n    enable (require.js, line 1177)\n    init (require.js, line 783)\n    callGetModule (require.js, line 1204)\n    completeLoad (require.js, line 1583)\n    onScriptLoad (require.js, line 1711)\n[Log] Loading extension: cite2c/main (main.min.js, line 12109)\n[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (backbone-min.map, line 0)\n```\n",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/181818074",
    "html_url": "https://github.com/jupyter/notebook/issues/1030#issuecomment-181818074",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1030",
    "id": 181818074,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE4MTgxODA3NA==",
    "user": {
      "login": "AGS-Knight",
      "id": 2307497,
      "node_id": "MDQ6VXNlcjIzMDc0OTc=",
      "avatar_url": "https://avatars.githubusercontent.com/u/2307497?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/AGS-Knight",
      "html_url": "https://github.com/AGS-Knight",
      "followers_url": "https://api.github.com/users/AGS-Knight/followers",
      "following_url": "https://api.github.com/users/AGS-Knight/following{/other_user}",
      "gists_url": "https://api.github.com/users/AGS-Knight/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/AGS-Knight/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/AGS-Knight/subscriptions",
      "organizations_url": "https://api.github.com/users/AGS-Knight/orgs",
      "repos_url": "https://api.github.com/users/AGS-Knight/repos",
      "events_url": "https://api.github.com/users/AGS-Knight/events{/privacy}",
      "received_events_url": "https://api.github.com/users/AGS-Knight/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-02-09T11:06:38Z",
    "updated_at": "2016-02-09T11:06:38Z",
    "author_association": "NONE",
    "body": "I've run notebooks in both Firefox and Safari and  am having the same issue.  Setting `c.Session.debug = True` doesn't display any information regarding this problem\n",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/226461304",
    "html_url": "https://github.com/jupyter/notebook/issues/1030#issuecomment-226461304",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1030",
    "id": 226461304,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIyNjQ2MTMwNA==",
    "user": {
      "login": "minrk",
      "id": 151929,
      "node_id": "MDQ6VXNlcjE1MTkyOQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/151929?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/minrk",
      "html_url": "https://github.com/minrk",
      "followers_url": "https://api.github.com/users/minrk/followers",
      "following_url": "https://api.github.com/users/minrk/following{/other_user}",
      "gists_url": "https://api.github.com/users/minrk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/minrk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/minrk/subscriptions",
      "organizations_url": "https://api.github.com/users/minrk/orgs",
      "repos_url": "https://api.github.com/users/minrk/repos",
      "events_url": "https://api.github.com/users/minrk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/minrk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-06-16T11:40:46Z",
    "updated_at": "2016-06-16T11:40:46Z",
    "author_association": "MEMBER",
    "body": "What version of the notebook is this, and how was it installed? Can you run the notebook without any extensions to see if an extension is the cause?\n",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/247136790",
    "html_url": "https://github.com/jupyter/notebook/issues/1030#issuecomment-247136790",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1030",
    "id": 247136790,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI0NzEzNjc5MA==",
    "user": {
      "login": "gnestor",
      "id": 512354,
      "node_id": "MDQ6VXNlcjUxMjM1NA==",
      "avatar_url": "https://avatars.githubusercontent.com/u/512354?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/gnestor",
      "html_url": "https://github.com/gnestor",
      "followers_url": "https://api.github.com/users/gnestor/followers",
      "following_url": "https://api.github.com/users/gnestor/following{/other_user}",
      "gists_url": "https://api.github.com/users/gnestor/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/gnestor/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/gnestor/subscriptions",
      "organizations_url": "https://api.github.com/users/gnestor/orgs",
      "repos_url": "https://api.github.com/users/gnestor/repos",
      "events_url": "https://api.github.com/users/gnestor/events{/privacy}",
      "received_events_url": "https://api.github.com/users/gnestor/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-09-14T20:05:42Z",
    "updated_at": "2016-09-14T20:05:42Z",
    "author_association": "CONTRIBUTOR",
    "body": "@AGS-Knight If you're still running into this issue, [upgrade notebook](https://github.com/jupyter/jupyter/blob/master/docs/source/projects/upgrade-notebook.rst) and try again.\n",
    "performed_via_github_app": null
  }
]
